
# [CVE-2021-3737](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-3737)

## Description

- `A flaw was found in python. An improperly handled HTTP response in the HTTP client code of python may allow a remote attacker, who controls the HTTP server, to make the client script enter an infinite loop, consuming CPU time. The highest threat from this vulnerability is to system availability.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 7.1

## Scores

- **Exploitability Score**:
  - 8.6
- **Impact Score**:
  - 6.9
- **Base Severity**:
  - HIGH

## Other Information

- **Publish Date**:
  - 2022-03-04 19:15:08
- **Vulnerability Status**:
  - Modified

## References

- **secalert@redhat.com**: https://bugs.python.org/issue44022
- **secalert@redhat.com**: https://bugzilla.redhat.com/show_bug.cgi?id=1995162
- **secalert@redhat.com**: https://github.com/python/cpython/pull/25916
- **secalert@redhat.com**: https://github.com/python/cpython/pull/26503
- **secalert@redhat.com**: https://lists.debian.org/debian-lts-announce/2023/05/msg00024.html
- **secalert@redhat.com**: https://lists.debian.org/debian-lts-announce/2023/06/msg00039.html
- **secalert@redhat.com**: https://python-security.readthedocs.io/vuln/urllib-100-continue-loop.html
- **secalert@redhat.com**: https://security.netapp.com/advisory/ntap-20220407-0009/
- **secalert@redhat.com**: https://ubuntu.com/security/CVE-2021-3737
- **secalert@redhat.com**: https://www.oracle.com/security-alerts/cpujul2022.html
